chocolatey/tools/installer.py

```diff
@@ -0,0 +1,388 @@
+#!/usr/bin/env python
```

Comment:
This seems to an alsost exact copy of the lektor installer. Even the Linux and macOS installer bits remain. What changes have been made? If there are little or no changes I think we should depend on the same file.

Reply:
So here's the change:

I've removed the part where the installer asks whether the lektor needs to be installed or not (Y/n)

This is because `chocolatey` provides for this step by default, and we don't have to ask the user again for it.